front/app/containers/AccessibilityStatement/index.tsx

```diff
@@ -44,11 +44,11 @@ const CookiePolicy = memo((props: WrappedComponentProps) => {
                     values={{
                       citizenLabLink: (
                         <a
-                          href="https://www.citizenlab.co/"
+                          href="https://www.govocal.co/"
                           target="_blank"
```

Comment:
@sebastienhoorens Just to confirm - this would be the new URL right? It's not `.com` now or anything? :thinking:

Reply:
@amanda-anderson I think it should be govocal.com (which works in my browser, while govocal.co doesn't), that's how I changed it in the sync branch at least.